.github/workflows/build.yaml

```diff
@@ -1,38 +1,38 @@
 name: Build and test
 
 on:
-  push:
-    branches:
-      - master
-  pull_request:
+    push:
+        branches:
+            - master
+    pull_request:
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Setup Node.js
-        uses: actions/setup-node@v4
-        with:
-          cache: 'yarn'
-
-      - name: Install NPM packages
-        run: yarn install --frozen-lockfile
-
-      - name: Check Linting Rules and Types
-        run: yarn lint
-
-      - name: test
-        run: yarn test --coverage
-
-      - name: Upload coverage
-        uses: actions/upload-artifact@v4
-        with:
-          name: coverage
-          path: |
-            coverage
-            !coverage/lcov-report
-
-      - name: build
-        run: yarn build
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Setup Node.js
+              uses: actions/setup-node@v4
+              with:
+                  cache: "yarn"
+
+            - name: Install NPM packages
+              run: yarn install --frozen-lockfile
+
+            - name: Check Linting Rules and Types
+              run: yarn lint
```

Comment:
There should also be a step for `yarn prettier:check` ([example](https://github.com/element-hq/element-call/blob/livekit/.github/workflows/lint.yaml#L18))